OpenSim/Analyses/JointReaction.cpp

```diff
@@ -188,24 +188,32 @@ setupProperties()
     _forcesFileNameProp.setName("forces_file");
     _forcesFileNameProp.setComment("The name of a file containing forces storage."
         "If a file name is provided, the applied forces for all actuators will be constructed "
-        "from the forces_file instead of from the states.  This option should be used "
+        "from the forces_file instead of from the states. This option should be used "
```

Comment:
"... the applied forces for all actuators will be constructed ..." -> "... the forces for all actuators will be applied according to values specified in the forces_file instead of being computed from the states."